tests/test_sr/sr-binary32.c

```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>
+#include <math.h>
+#include <assert.h>
+#include <stdlib.h>
+
+int main(int argc, char * argv[]){
+    assert(argc == 2);
+    int p = atoi(argv[1]);
+    float a = 1.25;
+    float b = pow(2, -p);
+
+    for (int i=0; i < 100; i++) {
```

Comment:
can we make the 100 a param and relates it to the tolerance define in the py script, both of them being determine in the run script at the same place?

Reply:
Ok.